### PR TITLE
Puts all job edits back in non-modular files and adjusts job titles.

### PR DESCRIPTION
**Pull request by GrayRachnid**
## About The Pull Request
This PR puts all job changes back in non-modular files and updates job titles to make more sense.
I understand that there is currently a design philosophy for each role so i've made sure that stays true despite my reservations for a few roles(like AI) that shouldn't have any.

Here goes.

Facility Director -> Captain | This isn't a facility anymore and is a moving ship, it only makes sense for the Captain to be referred as such. The Facility Director title has been removed.

Inventory Chief -> Removed | This title does not roll well off the tongue and in my opinion makes little sense. A better title would have been 'Inventory Keeper' but even that is stretching the duties of the QM.

Paracausel Scholar -> Moved from Chaplain to Librarian | A chaplain is there to offer religious advice, not explain to you paranormal science.

Director of Maintenance -> Engineering Director | Director of Maintenance is a title that makes absolutely no sense and is clearly just someone's attempt at adding Director to every head role. Nonetheless, i've made it something more tolerable.

Third Engineer -> Removed | Having this title implies that there is seniority in rank among engineers. I feel like it is acceptable to give people the option of choosing Junior ranks but Senior ranks should not be alt titles but rather full on different jobs.

Expedition Manager -> Expedition Leader | A manager isn't something the Pathfinder should be labelled.

Lead Scout -> Lead Pioneer | There's nothing wrong with Lead Scout but Lead Pioneer rolls easier off the tongue and meshes better with the Pioneer alt title of explorers.

Combat Pilot -> Removed | What are they flying, an A10 Warthog? No, they're just a normal pilot.

Exploration Director -> Expedition Director | The lead scientist would be leading the whole expedition mission of the ship, not just the tiny exploration department.

Regulatory Affairs Director -> Moved to become an alt title of IAA | The RD wouldn't be a regulatory affairs director, he'd be a lead scientist.

Field Technician -> Research Field Technician | Just making it easier to know what department they belong to, field technician is broad.

Senior Constable -> Added alt title for warden | It works well with Constable being available to normal security.

Investigator -> Criminal Investigator | Just something to clarify the alt title, bit more fluff.

Patrolman -> Removed | You're not walking the beat in NYC, this is a ship and you're a security officer on it. This title adjusts the expectations of security officers too much and does not mesh well with the setting.

Peacekeeper -> Removed | Does not mesh well with the setting. You're not on a peacekeeping mission to a vox world, you're working on a ship as an enforcer of corporate regulations.

Overseer: Digitised Deluxe Edition -> CPU | This title makes no sense and does not mesh well with the setting at all and is too long.

Ethical Compliance Analyst -> Regulatory Affairs Agent | As IAA you're not here to analyze ethics, you're there to make sure people follow regulations and be the oversight over heads, security and all departments.

=========================================

Any other edits that are not mentioned here were just spread out across different files that i've all localized in their non-modular counterparts.

## Why It's Good For The Game

Titles make a little bit more sense now and can be considered more setting friendly.

## Changelog
:cl:
tweak: altered job titles to make them more setting and common sense compliant.
code: removed all non-modular files that edited jobs and put everything in their non-modular counterparts.
/:cl:

### CODE REVIEW

**Comment by Captain277** [comment]
These all look fine except for moving Paracausal Scholar to the Librarian. That's not the intent or messaging behind the job title by any metric.

**Comment by GrayRachnid** [comment]
> 
> 
> These all look fine except for moving Paracausal Scholar to the Librarian. That's not the intent or messaging behind the job title by any metric.

Then what are the expectations and duties of a religious advisor Paracausel Scholar?

**Comment by Captain277** [comment]
It's a flavor title. Their job duties are the same. I appreciate the majority of these changes and the intent behind them, but I am 100% against moving this one.

**Comment by GrayRachnid** [comment]
> 
> 
> It's a flavor title. Their job duties are the same. I appreciate the majority of these changes and the intent behind them, but I am 100% against moving this one.

All of the titles i've removed can be considered flavor titles but some of these fluff titles change the interpretation of the jobs. I understand that you want to keep the title which is why i've moved it instead of removing it. A scholar is better suited as a fluff title for the Librarian, not the Chaplain. The chaplain is the advisor when it comes to religious affairs, not paranormal ones.

**Comment by Captain277** [comment]
The Chaplain is literally the advisor for paranormal affairs. That's why their gear hard counters paranormal events like wraiths, cults, and Wizards. When the Librarian can combat those threats, then I'll consider letting the title be moved to them. As it stands, the alt title fits the general role of the Chaplaincy. My stance hasn't changed.

**Comment by GrayRachnid** [comment]
The chaplain is the paranormal advisor only by process of elimination. No other role on the ship can perform 'holy' tasks like he can. Nonetheless, I don't disagree with you despite the fact that a Paracautel Scholar detaches the chaplain from their original tasks.

I'm going to keep the Paracautel Scholar title for librarians and give the chaplain an 'Exorcist' title to keep his religious roots relevant while giving him the credentials required to be a paranormal advisor.

**Comment by Captain277** [comment]
I mean, I've said several times now that I don't want the title moved. It isn't something that an entirely different title is going to suddenly change my mind on. There is no divorce from theological authority when someone is described as a Religious Scholar, and in terms of duty the two terms are interchangeable. While I can see why you're interpreting it your way, the point stands. I'm not willing to see the title moved to a job I don't feel it fits, and I'm not willing to compromise on it.

**Comment by GrayRachnid** [comment]
> 
> 
> I mean, I've said several times now that I don't want the title moved. It isn't something that an entirely different title is going to suddenly change my mind on. There is no divorce from theological authority when someone is described as a Religious Scholar, and in terms of duty the two terms are interchangeable. While I can see why you're interpreting it your way, the point stands. I'm not willing to see the title moved to a job I don't feel it fits, and I'm not willing to compromise on it.

A chaplain is meant to be religious, not scientific. From my understanding a Paracautel Scholar is more 'scientific' than religious or spiritual.

**Comment by Captain277** [comment]
Here. Let me help you out by putting it a different way.

**Comment by GrayRachnid** [comment]
Have it your way.

**Comment by GrayRachnid** [comment]
I've changed my mind. I'd still like this PR to go through and I am not budging on moving Paracausal Scholar to librarian. The Paracausal scholar is something of a scientific expert while the Chaplain is meant to be a religious/spiritual advisor. There's nothing wrong with a Spiritual Scholar being the chaplain but a Paracausal Scholar is not spiritual by any means.

**Comment by faaaay** [comment]
Plenty of religious figures take an interest in studying theology. Why is Spiritual Scholar alright, but not Paracausal Scholar?

**Comment by GrayRachnid** [comment]
> a Paracausal Scholar is not spiritual by any means.

**Comment by faaaay** [comment]
> > a Paracausal Scholar is not spiritual/religious by any means.

Isn't it? Most religions are inherently paracausal, revolving around the belief in beings or forces beyond our understanding of reality.

**Comment by GrayRachnid** [comment]
Causality is influence by which one event, process, state or object contributes to the production of another event, process, state or object where the cause is partly responsible for the effect, and the effect is partly dependent on the cause.

Paranormal events are purported phenomena described in popular culture, folk, and other non-scientific bodies of knowledge, whose existence within these contexts is described as beyond normal experience or scientific explanation.

I could not find Paracausal in an actual dictionary so I combined the two terms and formed my own understanding that Paracausal are Paranormal events that are caused by the butterfly effect. This is also reinforced by an event that Captain had ran the other day where the contents of the 'Paracausal' events were all centring around wormholes, space and magic.

**Comment by faaaay** [comment]
> 
> 
> Causality is influence by which one event, process, state or object contributes to the production of another event, process, state or object where the cause is partly responsible for the effect, and the effect is partly dependent on the cause.
> 
> Paranormal events are purported phenomena described in popular culture, folk, and other non-scientific bodies of knowledge, whose existence within these contexts is described as beyond normal experience or scientific explanation.
>
> I could not find Paracausal in an actual dictionary so I combined the two terms and formed my own understanding that Paracausal are Paranormal events that are caused by the butterfly effect. This is also reinforced by an event that Captain had ran the other day where the contents of the 'Paracausal' events were all centring around wormholes, space and magic.

I don't quite see how the para- prefix added to causality relates to the butterfly effect, especially given that the example event you gave doesn't have anything to do with it.

**Comment by GrayRachnid** [comment]
> 
> 
> > Causality is influence by which one event, process, state or object contributes to the production of another event, process, state or object where the cause is partly responsible for the effect, and the effect is partly dependent on the cause.
> > Paranormal events are purported phenomena described in popular culture, folk, and other non-scientific bodies of knowledge, whose existence within these contexts is described as beyond normal experience or scientific explanation.
> > I could not find Paracausal in an actual dictionary so I combined the two terms and formed my own understanding that Paracausal are Paranormal events that are caused by the butterfly effect. This is also reinforced by an event that Captain had ran the other day where the contents of the 'Paracausal' events were all centring around wormholes, space and magic.
> 
> I don't quite see how the para- prefix added to causality relates to the butterfly effect, especially given that the example event you gave doesn't have anything to do with it.

That's exactly my dilemma, I have no clue what the word actually means and how it relates to religious activities and the only source of information I could go after was an event that Captain hosted which seemed more scientific than religious in nature.

**Comment by GrayRachnid** [comment]
Also, Causality is essentially the butterfly effect.

**Comment by faaaay** [comment]
> > I don't quite see how the para- prefix added to causality relates to the butterfly effect, especially given that the example event you gave doesn't have anything to do with it.
> 
> That's exactly my dilemma, I have no clue what the word actually means and how it relates to religious activities and the only source of information I could go after was an event that Captain hosted which seemed more scientific than religious in nature. Thus, moved to librarian.

If you're not sure what it means, wouldn't it be reasonable to leave it out until it's certain?


> Also, Causality is essentially the butterfly effect.

Which would imply that paracausality deals with events that do not follow cause and effect.

**Comment by GrayRachnid** [comment]
> 
> 
> > > I don't quite see how the para- prefix added to causality relates to the butterfly effect, especially given that the example event you gave doesn't have anything to do with it.
> > 
> > 
> > That's exactly my dilemma, I have no clue what the word actually means and how it relates to religious activities and the only source of information I could go after was an event that Captain hosted which seemed more scientific than religious in nature. Thus, moved to librarian.
> 
> If you're not sure what it means, wouldn't it be reasonable to leave it out until it's certain?
> 
That's why i'm here, i'm trying to understand what it means but Captain preferred to mark my PR DNM rather than explain.

> > Also, Causality is essentially the butterfly effect.
> 
> Which would imply that paracausality deals with events that do not follow cause and effect.
The butterfly effect does depend on cause. I think we're talking on different wavelengths here. I say Casualty is essentially cause and effect because the butterfly effect has all the elements of causality if oversimplified. All you need in the butterfly effect is a cause and you'll get your effect somewhere down the line. Causality properly breaks down cause and effect rather than oversimplify it like the butterfly effect.

**Review by Captain277 — CHANGES_REQUESTED** [review]
This is a formal review. The Paracausal Scholar title does not belong under the Librarian job slot. Return it to the Chaplain job slot for PR approval.

**Comment by Captain277** [comment]
I admire that you're unwilling to budge on this, but I'm not either. Regardless of your naked contempt, this PR will not go through without the requested change. It's as simple as that and no debate is going to change it.

**Comment by Captain277** [comment]
Shit, well, for the record I didn't intend to directly change the files myself. I figured that was going to show the requested changes.

**Comment by GrayRachnid** [comment]
> 
> 
> This is a formal review. The Paracausal Scholar title does not belong under the Librarian job slot. Return it to the Chaplain job slot for PR approval.

Please explain to me how a scientific scholar title belongs to a religious/spiritual advisor position.

**Comment by Captain277** [comment]
I'm really not interested in constantly having this debate. It's a title that was assigned to the Chaplaincy. Your idea of what classifies as a scientific scholar title, or what the exact duties of the Chaplain entail are not in line with the lore or design intent currently in place on the server. Regardless of the logic you're intent on using to justify the change, this title reassignment is a deal breaker and will not pass review. There is no reason for it to be this big - or this enduring - of a concern. The judgement isn't changing, and it isn't even really up for discussion. I've requested the change, and without it, this PR is dead in the water.

**Comment by buffyuwu** [comment]
> > This is a formal review. The Paracausal Scholar title does not belong under the Librarian job slot. Return it to the Chaplain job slot for PR approval.
> 
> Please explain to me how a scientific scholar title belongs to a religious/spiritual advisor position.

You're intent on dying on this hill without even trying to understand _why_ it won't be changed. It's not a "scientific scholar" title, as you put it. It's a title with roots to the CCO PMD unit, namely the Paracausal Investigations/Paracausal Response teams. Paracausal Scholars are like liaisons in a way to these groups and it ties into _many_ event lines. They handle many occult responsibilities on-station and during events, and have for some time.

Captain explained to you in their third reply that it was a flavor title for Chaplain. As in, they fulfill the duties and expectations of the Chaplain. They explained that they handle all paranormal affairs, even things like fighting wraiths, and so on. From Cap's explanation that 'yes indeed this title belongs to a Chaplain', you persisted that this Chaplain title should be moved to Librarian, because of the word 'scholar'. The librarian catalogues and keeps books, and uses a barcode scanner. They aren't religious/occult scholars wielding holy blades, you know?

In closing, it is the job of the maintainer to ensure our codebase remains solid, and doesn't stray from our lore. Captain is well within their right to tell you if a title will or will not work for that purpose. For the future going forward, I recommend going into a discussion with the intention of _learning_ something from it, rather than asserting your will to be correct over others without taking the time to understand their perspective. 

You're a great developer and I hope that this was just a minor misunderstanding that you've grown clearer on through further explanation. 

tl;dr Paracausal Scholars belong on the Chaplain slot, both in their _function_ and code-wise to fulfill their duties both in and out of character. They're mage/warrior/scholarly-of-the-arts types that slay demons and cleanse things.

**Comment by GrayRachnid** [comment]
> 
> 
> > > This is a formal review. The Paracausal Scholar title does not belong under the Librarian job slot. Return it to the Chaplain job slot for PR approval.
> > 
> > 
> > Please explain to me how a scientific scholar title belongs to a religious/spiritual advisor position.
> 
> I want to help you understand _why_ it won't be changed. It's not a "scientific scholar" title, as you put it. It's a title with roots to the CCO PMD unit, namely the Paracausal Investigations/Paracausal Response teams. Paracausal Scholars are like liaisons in a way to these groups and it ties into _many_ event lines. They handle many occult responsibilities on-station and during events, and have for some time.
> 
> Captain explained to you in their third reply that it was a flavor title for Chaplain. As in, they fulfill the duties and expectations of the Chaplain. They explained that they handle all paranormal affairs, even things like fighting wraiths, and so on. From Cap's explanation that 'yes indeed this title belongs to a Chaplain', you persisted that this Chaplain title should be moved to Librarian, because of the word 'scholar'. The librarian catalogues and keeps books, and uses a barcode scanner. They aren't religious/occult scholars wielding holy blades, you know?
> 
> In closing, it is the job of the maintainer to ensure our codebase remains solid, and doesn't stray from our lore. Captain is well within their right to tell you if a title will or will not work for that purpose. For the future going forward, I recommend going into a discussion with the intention of _learning_ something from it, rather than asserting your will to be correct over others without taking the time to understand their perspective.
> 
> You're a great developer and I hope that this was just a minor misunderstanding that you've grown clearer on through further explanation.
> 
> tl;dr Paracausal Scholars belong on the Chaplain slot, both in their _function_ and code-wise to fulfill their duties both in and out of character. They're mage/warrior/scholarly-of-the-arts types that slay demons and cleanse things.

And i've said again that it's not "scholar" that makes it a chaplain title, but Paracausel.

All i'm asking for is an explanation of how a scientific(Paracausal) scholar is appropriate for a religious/spiritual role. As i've said in a previous comment, there's nothing wrong with the chaplain having a Theology/Spiritual scholar. What bothers me is the scientific role that Paracausal gives the chaplain. Paranormal or not, Paracausal leans towards more science than religion/spiritual.

If you can explain to me why Paracausal, a scientific term, is being used to alter the title(and therefore the assumed duties) of the chaplain then I am more than willing to accommodate to your request and adjust my PR. I just want an explanation, I don't want to fight you.

**Comment by faaaay** [comment]
> If you can explain to me why Paracausal, a scientific term, is being used to alter the title(and therefore the assumed duties) of the chaplain then I am more than willing to accommodate to your request and adjust my PR. I just want an explanation, I don't want to fight you.

There are quite a few alternate titles that change the flavour, background and general vibe of a job. A Maid may be mechanically identical to a Viscera Cleaner, but there's a world of difference. Would it be so bad to allow that for Chaplains?

**Comment by GrayRachnid** [comment]
> 
> 
> > If you can explain to me why Paracausal, a scientific term, is being used to alter the title(and therefore the assumed duties) of the chaplain then I am more than willing to accommodate to your request and adjust my PR. I just want an explanation, I don't want to fight you.
> 
> There are quite a few alternate titles that change the flavour, background and general vibe of a job. A Maid may be mechanically identical to a Viscera Cleaner, but there's a world of difference. Would it be so bad to allow that for Chaplains?

A maid and a viscera cleaner both serve the same role at the end of the day, cleaning a mess. A chaplain is a religious/spiritual advisor. A paracausel scholar is a *scientific* expert.

**Comment by faaaay** [comment]
> A maid and a viscera cleaner both serve the same role at the end of the day, cleaning a mess. A chaplain is a religious/spiritual advisor. A paracausel scholar is a _scientific_ expert.

They may not share the duty of counselling, but they _do_ both have a role that buffy highlighted - protecting the station from threats beyond our understanding.

**Comment by GrayRachnid** [comment]
> 
> 
> > A maid and a viscera cleaner both serve the same role at the end of the day, cleaning a mess. A chaplain is a religious/spiritual advisor. A paracausel scholar is a _scientific_ expert.
> 
> They may not share the duty of counselling, but they _do_ both have a role that buffy highlighted - protecting the station from threats beyond our understanding.

Then how about a compromise of giving the Librarian a null-rod copy that allows them to perform the same roles?
They don't have much of a role currently and i'm sure two paranormal experts is better than one.

**Comment by buffyuwu** [comment]
> > > > This is a formal review. The Paracausal Scholar title does not belong under the Librarian job slot. Return it to the Chaplain job slot for PR approval.
> > > 
> > > 
> > > Please explain to me how a scientific scholar title belongs to a religious/spiritual advisor position.
> > 
> > 
> > I want to help you understand _why_ it won't be changed. It's not a "scientific scholar" title, as you put it. It's a title with roots to the CCO PMD unit, namely the Paracausal Investigations/Paracausal Response teams. Paracausal Scholars are like liaisons in a way to these groups and it ties into _many_ event lines. They handle many occult responsibilities on-station and during events, and have for some time.
> > Captain explained to you in their third reply that it was a flavor title for Chaplain. As in, they fulfill the duties and expectations of the Chaplain. They explained that they handle all paranormal affairs, even things like fighting wraiths, and so on. From Cap's explanation that 'yes indeed this title belongs to a Chaplain', you persisted that this Chaplain title should be moved to Librarian, because of the word 'scholar'. The librarian catalogues and keeps books, and uses a barcode scanner. They aren't religious/occult scholars wielding holy blades, you know?
> > In closing, it is the job of the maintainer to ensure our codebase remains solid, and doesn't stray from our lore. Captain is well within their right to tell you if a title will or will not work for that purpose. For the future going forward, I recommend going into a discussion with the intention of _learning_ something from it, rather than asserting your will to be correct over others without taking the time to understand their perspective.
> > You're a great developer and I hope that this was just a minor misunderstanding that you've grown clearer on through further explanation.
> > tl;dr Paracausal Scholars belong on the Chaplain slot, both in their _function_ and code-wise to fulfill their duties both in and out of character. They're mage/warrior/scholarly-of-the-arts types that slay demons and cleanse things.
> 
> And i've said again that it's not "scholar" that makes it a chaplain title, but Paracausel.
> 
> All i'm asking for is an explanation of how a scientific(Paracausal) scholar is appropriate for a religious/spiritual role. As i've said in a previous comment, there's nothing wrong with the chaplain having a Theology/Spiritual scholar. What bothers me is the scientific role that Paracausal gives the chaplain. Paranormal or not, Paracausal leans towards more science than religion/spiritual.
> 
> If you can explain to me why Paracausal, a scientific term, is being used to alter the title(and therefore the assumed duties) of the chaplain then I am more than willing to accommodate to your request and adjust my PR. I just want an explanation, I don't want to fight you.

As stated above, Paracausal is the name of a unit that handles paranormal affairs. There is little, if any, roots in the role being used in a scientific capacity outside of maybe their documentation of such affairs. The word 'Paracausal' is **ambiguous**, and has no solid place in any real-life scientific space nor is it a commonly used word. It's rise to popularity came from Bungie's Destiny games, and the definition used in those games is the same used in our universe. 

Further, even if a word is based in reality (this one specifically is not) it can be used for different things inside of video games, stories, and sci-fi. That's why it's called science _fiction_, after all.

Hope this helps!

**Comment by faaaay** [comment]
>
> As stated above, Paracausal is the name of a unit that handles paranormal affairs. There is little, if any, roots in the role being used in a scientific capacity outside of maybe their documentation of such affairs. **The word 'Paracausal' is **ambiguous**, and has no solid place in any real-life scientific space nor is it a commonly used word.** It's rise to popularity came from Bungie's Destiny games, and the definition used in those games is the same used in our universe.
> 
> Further, even if a word is based in reality (this one specifically is not) it can be used for different things inside of video games, stories, and sci-fi. That's why it's called science _fiction_, after all.
> 
> Hope this helps!

This right here is the key. At the end of the day, it's technobabble, like calling a Field Medic a "Bluespace Recombobulator". The term has no real meaning outside of fiction, so we're not restricted in how we use it.

**Comment by GrayRachnid** [comment]
> 
> 
> > > > > This is a formal review. The Paracausal Scholar title does not belong under the Librarian job slot. Return it to the Chaplain job slot for PR approval.
> > > > 
> > > > 
> > > > Please explain to me how a scientific scholar title belongs to a religious/spiritual advisor position.
> > > 
> > > 
> > > I want to help you understand _why_ it won't be changed. It's not a "scientific scholar" title, as you put it. It's a title with roots to the CCO PMD unit, namely the Paracausal Investigations/Paracausal Response teams. Paracausal Scholars are like liaisons in a way to these groups and it ties into _many_ event lines. They handle many occult responsibilities on-station and during events, and have for some time.
> > > Captain explained to you in their third reply that it was a flavor title for Chaplain. As in, they fulfill the duties and expectations of the Chaplain. They explained that they handle all paranormal affairs, even things like fighting wraiths, and so on. From Cap's explanation that 'yes indeed this title belongs to a Chaplain', you persisted that this Chaplain title should be moved to Librarian, because of the word 'scholar'. The librarian catalogues and keeps books, and uses a barcode scanner. They aren't religious/occult scholars wielding holy blades, you know?
> > > In closing, it is the job of the maintainer to ensure our codebase remains solid, and doesn't stray from our lore. Captain is well within their right to tell you if a title will or will not work for that purpose. For the future going forward, I recommend going into a discussion with the intention of _learning_ something from it, rather than asserting your will to be correct over others without taking the time to understand their perspective.
> > > You're a great developer and I hope that this was just a minor misunderstanding that you've grown clearer on through further explanation.
> > > tl;dr Paracausal Scholars belong on the Chaplain slot, both in their _function_ and code-wise to fulfill their duties both in and out of character. They're mage/warrior/scholarly-of-the-arts types that slay demons and cleanse things.
> > 
> > 
> > And i've said again that it's not "scholar" that makes it a chaplain title, but Paracausel.
> > All i'm asking for is an explanation of how a scientific(Paracausal) scholar is appropriate for a religious/spiritual role. As i've said in a previous comment, there's nothing wrong with the chaplain having a Theology/Spiritual scholar. What bothers me is the scientific role that Paracausal gives the chaplain. Paranormal or not, Paracausal leans towards more science than religion/spiritual.
> > If you can explain to me why Paracausal, a scientific term, is being used to alter the title(and therefore the assumed duties) of the chaplain then I am more than willing to accommodate to your request and adjust my PR. I just want an explanation, I don't want to fight you.
> 
> As stated above, Paracausal is the name of a unit that handles paranormal affairs. There is little, if any, roots in the role being used in a scientific capacity outside of maybe their documentation of such affairs. The word 'Paracausal' is **ambiguous**, and has no solid place in any real-life scientific space nor is it a commonly used word. It's rise to popularity came from Bungie's Destiny games, and the definition used in those games is the same used in our universe.
> 
> Further, even if a word is based in reality (this one specifically is not) it can be used for different things inside of video games, stories, and sci-fi. That's why it's called science _fiction_, after all.
> 
> Hope this helps!

It unfortunately has not helped as you continue to reinforce my belief that a Paracausel Scholar is a scientific expert and not a religious/spiritual one. 

I'll re-state that the few times i've seen the word used in Captain's events have lead me to the belief that it is the study of outside-of-the-norm causality in space. This can go from making portals in space to generating wormholes or what have you. Essentially it boils down to 'things we do not understand but have an actual cause-and-effect' which is scientific no matter what way you look at it.

The chaplain is a religious and spiritual advisor, not a scientific advisor.



> 
> 
> > As stated above, Paracausal is the name of a unit that handles paranormal affairs. There is little, if any, roots in the role being used in a scientific capacity outside of maybe their documentation of such affairs. **The word 'Paracausal' is **ambiguous**, and has no solid place in any real-life scientific space nor is it a commonly used word.** It's rise to popularity came from Bungie's Destiny games, and the definition used in those games is the same used in our universe.
> > Further, even if a word is based in reality (this one specifically is not) it can be used for different things inside of video games, stories, and sci-fi. That's why it's called science _fiction_, after all.
> > Hope this helps!
> 
> This right here is the key. At the end of the day, it's technobabble, like calling a Field Medic a "Bluespace Recombobulator". The term has no real meaning outside of fiction, so we're not restricted in how we use it.

If you consider it reasonable to call a field medic a bluespace recombobulator then why has it not been done yet?

**Comment by Captain277** [comment]
> Then how about a compromise of giving the Librarian a null-rod copy that allows them to perform the same roles?
> They don't have much of a role currently and i'm sure two paranormal experts is better than one.

Advocating for turning the Librarian into a Chaplain clone just to maintain this desire to move the job title to them is frankly comical. There doesn't need to be a compromise here. The title belonged with the Chaplain from the start, and you're asking me to create *another Chaplain* just so it can be moved there. I'm just gonna take that as a tacit concession of the point, honestly.

**Comment by GrayRachnid** [comment]
> 
> 
> > Then how about a compromise of giving the Librarian a null-rod copy that allows them to perform the same roles?
> > They don't have much of a role currently and i'm sure two paranormal experts is better than one.
> 
> Advocating for turning the Librarian into a Chaplain clone just to maintain this desire to move the job title to them is frankly comical. There doesn't need to be a compromise here. The title belonged with the Chaplain from the start, and you're asking me to create _another Chaplain_ just so it can be moved there. I'm just gonna take that as a tacit concession of the point, honestly.

Please do not put words in my mouth. I specifically stated that it would never hurt to have two paranormal experts. Since your events seem to focus on the paranormal, I don't see why you can't have a logical-scientific expert and a more religious-spiritual expert at the same time.

**Comment by Captain277** [comment]
I'm not putting words in your mouth. You literally just asked me to turn the Librarian into a Chaplain because you don't want to admit the title doesn't need to be moved.

**Comment by Captain277** [comment]
Anyways, it's pretty clear that we're at an impasse here. I'm not going to let this through as it is, and you're clearly not intent on changing it.

**Comment by GrayRachnid** [comment]
> 
> 
> I'm not putting words in your mouth. You literally just asked me to turn the Librarian into a Chaplain because you don't want to admit the title doesn't need to be moved.

You're the one refusing to admit that the title doesn't want to be moved, i'm the one insisting that it be removed from Chaplain as it is not a religious flavor title and you have yet to speak to me directly as to why it should exist.

**Comment by buffyuwu** [comment]
> > I'm not putting words in your mouth. You literally just asked me to turn the Librarian into a Chaplain because you don't want to admit the title doesn't need to be moved.
> 
> You're the one refusing to admit that the title doesn't want to be moved, i'm the one insisting that it be removed from Chaplain as it is not a religious flavor title and you have yet to speak to me directly as to why it should exist.

I thought I explained it really well, so I recommend reading my replies to you in this thread.

**Comment by GrayRachnid** [comment]
> 
> 
> > > I'm not putting words in your mouth. You literally just asked me to turn the Librarian into a Chaplain because you don't want to admit the title doesn't need to be moved.
> > 
> > 
> > You're the one refusing to admit that the title doesn't want to be moved, i'm the one insisting that it be removed from Chaplain as it is not a religious flavor title and you have yet to speak to me directly as to why it should exist.
> 
> I thought I explained it really well, so I recommend reading my replies to you in this thread.

And I recommend reading mine as i'm still not convinced how this is not a scientific title. All your replies have only amplified my belief that it is.

**Comment by buffyuwu** [comment]
> > > > I'm not putting words in your mouth. You literally just asked me to turn the Librarian into a Chaplain because you don't want to admit the title doesn't need to be moved.
> > > 
> > > 
> > > You're the one refusing to admit that the title doesn't want to be moved, i'm the one insisting that it be removed from Chaplain as it is not a religious flavor title and you have yet to speak to me directly as to why it should exist.
> > 
> > 
> > I thought I explained it really well, so I recommend reading my replies to you in this thread.
> 
> And I recommend reading mine as i'm still not convinced how this is not a scientific title. All your replies have only amplified my belief that it is.

What makes you believe that our occult scholar role is scientific? Could you provide an example?

**Comment by GrayRachnid** [comment]
> 
> 
> > > > > I'm not putting words in your mouth. You literally just asked me to turn the Librarian into a Chaplain because you don't want to admit the title doesn't need to be moved.
> > > > 
> > > > 
> > > > You're the one refusing to admit that the title doesn't want to be moved, i'm the one insisting that it be removed from Chaplain as it is not a religious flavor title and you have yet to speak to me directly as to why it should exist.
> > > 
> > > 
> > > I thought I explained it really well, so I recommend reading my replies to you in this thread.
> > 
> > 
> > And I recommend reading mine as i'm still not convinced how this is not a scientific title. All your replies have only amplified my belief that it is.
> 
> What makes you believe that our occult scholar role is scientific? Could you provide an example?

Two days ago, 'paracausel activity' was detected on the ship and by the words of what I believe was Captain's event character all the paracausel activity was wormholes and space anomalies. Nothing occult about that.

**Comment by buffyuwu** [comment]
> > > > > > I'm not putting words in your mouth. You literally just asked me to turn the Librarian into a Chaplain because you don't want to admit the title doesn't need to be moved.
> > > > > 
> > > > > 
> > > > > You're the one refusing to admit that the title doesn't want to be moved, i'm the one insisting that it be removed from Chaplain as it is not a religious flavor title and you have yet to speak to me directly as to why it should exist.
> > > > 
> > > > 
> > > > I thought I explained it really well, so I recommend reading my replies to you in this thread.
> > > 
> > > 
> > > And I recommend reading mine as i'm still not convinced how this is not a scientific title. All your replies have only amplified my belief that it is.
> > 
> > 
> > What makes you believe that our occult scholar role is scientific? Could you provide an example?
> 
> Two days ago, 'paracausel activity' was detected on the ship and by the words of what I believe was Captain's event character all the paracausel activity was wormholes and space anomalies. Nothing occult about that.

When it concerns things that are little-understood in-character, it's easy to jump to conclusions. Have you considered that they're not run-of-the-mill wormholes if command has called in the Paracausal investigators/response team? You're expected to maintain that this is a universe in which you won't always know what is happening, when it is happening. I encourage you to participate more in-game on the RP server to see what the Paracausal lore is all about, and to consider that experiencing a single event may be a piece of, or independent from, an overall story that concerns a larger plot or subject matter.

**Comment by faaaay** [comment]
> Two days ago, 'paracausel activity' was detected on the ship and by the words of what I believe was Captain's event character all the paracausel activity was wormholes and space anomalies. Nothing occult about that.

And yet here's Captain, telling you that paracausality relates to the occult.
